app/controllers/api/functions.php

```diff
@@ -569,6 +675,15 @@
         $results = $dbForProject->find('tags', $queries, $limit, $offset, [], [$orderType], $cursorTag ?? null, $cursorDirection);
         $sum = $dbForProject->count('tags', $queries, APP_LIMIT_COUNT);
 
+        // Get Current Build Data
+        foreach ($results as &$tag) {
```

Comment:
Again, I think we can skip this output and include it in the console from a second API call.